src/PKSim.Infrastructure/Services/LazyLoadTask.cs

```diff
@@ -113,14 +113,21 @@ private void loadSimulations(Simulation simulation)
 
          //updating results may triggered update of has changed flag that is not accurate. We save the original state and update it at the end
          var hasChanged = simulation.HasChanged;
-         
+
          //Only load results for individual simulations
          if (simulation.IsAnImplementationOf<IndividualSimulation>())
             _simulationResultsLoader.LoadResultsFor(simulation.DowncastTo<IndividualSimulation>());
 
          else if (simulation.IsAnImplementationOf<PopulationSimulation>())
             _simulationAnalysesLoader.LoadAnalysesFor(simulation.DowncastTo<PopulationSimulation>());
 
+         //make sure each individual gets the expression profile defined in the simulation)  
+         var simulationSubject = simulation.BuildingBlock<ISimulationSubject>();
+       
+         //this can happen for an imported simulation
+         if (simulationSubject != null)
```

Comment:
When we load a simulation , we add the references  (individual in the simulation used expression profile in the simulation)